File: Semestrul 4/MPP/Proiecte/java_gui/src/main/java/map/repository/database/DonatorRepo0.java
package map.repository.database;

import map.domain.Donator;
import map.repository.Repository;

import java.util.List;

public interface DonatorRepo0 extends Repository<Donator, Integer> {
    List<Donator> findByNameContaining(String partialName);

    Donator findByFullName(String numeDonator);
}

File: Semestrul 4/MPP/Proiecte/c#_gui/repository/DonatorRepo0.cs
using System.Collections.Generic;
using csharp_avalonia.domain;

namespace csharp_avalonia.repository;

public interface DonatorRepo0 : Repository<Donator, int>
{
    List<Donator> FindByNameContaining(string partialName);
    Donator FindByFullName(string donorName);
}